great_expectations/core/__init__.py

```diff
@@ -1147,6 +1147,54 @@ class ExpectationConfiguration(DictDot):
                 "catch_exceptions": False,
             },
         },
+        "_expect_column_values_to_be_of_type__aggregate": {
+            "domain_kwargs": ["column", "row_condition", "condition_parser"],
+            "success_kwargs": ["type_", "mostly"],
+            "default_kwarg_values": {
+                "row_condition": None,
+                "condition_parser": "pandas",
+                "mostly": None,
+                "result_format": "BASIC",
+                "include_config": True,
+                "catch_exceptions": False,
+            },
+        },
```

Comment:
@jcampbell - do we actually want these here? Added them because I noticed a bunch of KeyErrors when running tests. Which brought up another question - don't we need to add some machinery for handling custom expectations?

Reply:
Do you mean the full range of default kwargs?

I think for the question of custom expectations, you're absolutely right--we need to make it possible to have a default default value here. Following principle of "it's better if not perfect", I think the overall defaults will be sufficient.